lib/arjdbc/abstract/connection_management.rb

```diff
@@ -0,0 +1,22 @@
+module ArJdbc
+  module Abstract
+    module ConnectionManagement
+
+      def active?
+        @connection.active?
+      end
+
+      def disconnect!
+        super
+        @connection.disconnect!
+      end
+
+      def reconnect!
+        super
+        @connection.reconnect! # handles adapter.configure_connection
```

Comment:
Doesn't this return @connection?

Reply:
I don't know, this was another piece I copied over from the jdbc adapter, I'll see if that is what it is doing and update it if it does return it

Reply:
All the references to this that I could find didn't seem to expect a response, so I made it so this no longer returns the connection